hawc/apps/epiv2/templates/epiv2/design_detail.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'assessment-rooted.html' %}
+
+{% block content %}
+<h1 class="d-inline-block">{{object.name}}</h1>
+{% if obj_perms.edit and crud == "Read" %}
+<div class="dropdown btn-group float-right">
+    <a class="btn btn-primary dropdown-toggle" data-toggle="dropdown">Actions</a>
+    <div class="dropdown-menu dropdown-menu-right">
+        <span class="dropdown-header">Study design editing</span>
+        <a class="dropdown-item" href="{% url 'epiv2:des_update' object.pk %}">Update</a>
+        <a class="dropdown-item" href="{% url 'epiv2:des_delete' pk=object.pk %}">Delete</a>
+    </div>
+</div>
+{% endif %}
+<div id="objContainer"></div>
```

Comment:
delete objContainer for now; we can add later when we have some javascript that builds this